gnovm/pkg/benchops/exporter.go

```diff
@@ -31,8 +31,9 @@ type exporter struct {
 // export code, duration, size in a 10 bytes record
 // byte 1: OpCode
 // byte 2: StoreCode
-// byte 3-6: Duration
-// byte 7-10: Size
+// byte 3: NativeCode
```

Comment:
What if we just make the code two-byte, and don't restrict it to the store?

Honestly it could even be a `uint32`.